### PR TITLE
Features/add deployment page

### PR DESCRIPTION
**Pull request by Redm4x**
- Add deployment page
- Add link to deployment page in msgs
- Wrap block/tx/msg insertion into a transaction

### CODE REVIEW

File: web/src/components/shared/TxMessages/TxMessageRow.tsx [hunk 1]
```diff
@@ -241,9 +241,9 @@ const TxMessage: React.FunctionComponent<TxMessageProps> = ({ message }) => {
       return <MsgDeleteProvider_v2 message={message} />;
     case "/akash.audit.v1beta2.MsgDeleteProviderAttributes": // 0E708A727028E976954B41C2B3E52D08246082EF6031C9BE629D9F8C7E037F70
       return <MsgDeleteProviderAttributes_v2 message={message} />;
-    case "/akash.deployment.v1beta2.MsgDepositDeployment": // 799C4904CC094FCDF8F538A35B16307C801F516356A2DD7912C2C7F7EC436507
+    case "/akash.deployment.v1beta2.MsgDepositDeployment": // 9765F5F540AD14BD677E238A9CD50D1F42CE0EE32EE9DDB267C5C6AC87DFEC86
       return <MsgDepositDeployment_v2 message={message} />;
-    case "/akash.deployment.v1beta2.MsgPauseGroup": // AE8751F67C2D1380550308A7F9B68177F204D0BA6DE638C1898469ACB3C83FA4
+    case "/akash.deployment.v1beta2.MsgPauseGroup":
```
Comment (baktun14) [inline web/src/components/shared/TxMessages/TxMessageRow.tsx:246]: You removed this tx hash?

File: web/src/components/shared/TxMessages/akash/v1beta1/MsgCreateBid.tsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ type TxMessageProps = {
 };
 
 export const MsgCreateBid: React.FunctionComponent<TxMessageProps> = ({ message }) => {
+  console.log(message);
```
Comment (baktun14) [inline web/src/components/shared/TxMessages/akash/v1beta1/MsgCreateBid.tsx:13]: :scissors:

File: web/src/pages/deployments/[owner]/[dseq].tsx [hunk 3]
```diff
@@ -0,0 +1,231 @@
+import Paper from "@mui/material/Paper";
```
Comment (baktun14) [inline web/src/pages/deployments/[owner]/[dseq].tsx:1]: Is there a reason we don't put directly `/deployments/[dseq]` without the owner?

File: web/src/components/shared/TxMessages/TxMessageRow.tsx (hunk 1 above)
Comment (Redm4x) [reply web/src/components/shared/TxMessages/TxMessageRow.tsx:246]: It wasn't a `v1beta2` type and I could not find any

Comment (baktun14) [reply web/src/components/shared/TxMessages/TxMessageRow.tsx:246]: Ah ok good

File: web/src/pages/deployments/[owner]/[dseq].tsx (hunk 3 above)
Comment (Redm4x) [reply web/src/pages/deployments/[owner]/[dseq].tsx:1]: As far as I know `dseq` are not unique across the blockchain, they are unique across an account and we need the owner to differentiate different deployment with the same dseq.

Comment (baktun14) [reply web/src/pages/deployments/[owner]/[dseq].tsx:1]: Ah ok that's what I was thinking. Sounds good then.

**Review by baktun14 — APPROVED** [review]

**Pull request merged**